src/gt4py/gtscript.py

```diff
@@ -260,13 +270,17 @@ def stencil(
         time_keys = ("parse_time", "module_time", "codegen_time", "build_time", "load_time")
         build_info.update({time_key: 0.0 for time_key in time_keys})
 
+    if cache_settings is None:
+        cache_settings = {}
+
     build_options = gt_definitions.BuildOptions(
         name=name,
         module=module,
         format_source=format_source,
         rebuild=rebuild,
         backend_opts=kwargs,
         build_info=build_info,
+        cache_settings=cache_settings,
```

Comment:
Have you considered inlining the if block above?
```suggestion
        cache_settings=cache_settings or {},
```